cmd/main.go

```diff
@@ -155,9 +168,9 @@ func main() {
 		TestProfilesPersister:  testConfigPersister,
 		GenericPersister:       dbHandler,
 		GraphqlHandler: graphql.New(graphql.Options{
-			DBHandler:        &dbHandler,
-			GetKubeClient:    helpers.NewKubeClientGenerator(viper.GetString("KUBECONFIG_FOLDER")),
-			GetDynamicClient: helpers.NewDynamicClientGenerator(viper.GetString("KUBECONFIG_FOLDER")),
+			Logger:     log,
+			DBHandler:  &dbHandler,
+			KubeClient: &kubeclient,
 		}),
 		GraphqlPlayground: graphql.NewPlayground(graphql.Options{
```

Comment:
I still haven't seen documentation on why this is included; how it is protected.

Reply:
Protection - They're done the usual way, how we do it for REST handlers.
What exactly to document in here @leecalcote ?
There are included here because resolver implementations have dependency on these.